Bottom: 529afa68edd8eef7ea387ac0f3f8507492f8b1ec
Top:    18b6278412bd6b899f02a2b50b08519f31beb794
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 16:47:11 +0100

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 4594c63..d2f49ec 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -635,10 +635,11 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale
 
   global_scale = scale;
 
-  if (poly->Clipped == NULL) {
-    fprintf (stderr, "hidgl_fill_pcb_polygon: poly->Clipped == NULL\n");
-    return;
-  }
+  if (poly->Clipped == NULL)
+    {
+      fprintf (stderr, "hidgl_fill_pcb_polygon: poly->Clipped == NULL\n");
+      return;
+    }
 
   /* TODO: Just draw our triangles, no need to flush the buffer */
   hidgl_flush_triangles (&buffer);
